2014-02-10T07:36:12.116557+00:00 app[web.1]:   File "/app/rosterrun.py", line 330, in <module>
2014-02-10T07:36:12.116557+00:00 app[web.1]:     @app.route('/treasury', methods=['GET', 'POST'])
2014-02-10T07:36:12.116557+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 62, in wrapper_func
2014-02-10T07:36:12.116557+00:00 app[web.1]:     return f(self, *args, **kwargs)
2014-02-10T07:36:12.116732+00:00 app[web.1]: AssertionError: View function mapping is overwriting an existing endpoint function: market_results
2014-02-10T07:36:12.116732+00:00 app[web.1]: Traceback (most recent call last):
2014-02-10T07:36:12.116732+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 495, in spawn_worker
2014-02-10T07:36:12.116732+00:00 app[web.1]:     worker.init_process()
2014-02-10T07:36:12.116557+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1013, in decorator
2014-02-10T07:36:12.116557+00:00 app[web.1]:     self.add_url_rule(rule, endpoint, f, **options)
2014-02-10T07:36:12.116732+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 984, in add_url_rule
2014-02-10T07:36:12.116732+00:00 app[web.1]:     'existing endpoint function: %s' % endpoint)
2014-02-10T07:36:12.116732+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/workers/base.py", line 106, in init_process
2014-02-10T07:36:12.116732+00:00 app[web.1]:     self.callable = self.load()
2014-02-10T07:36:12.117462+00:00 app[web.1]:     return self.load_wsgiapp()
2014-02-10T07:36:12.117462+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 49, in load_wsgiapp
2014-02-10T07:36:12.117462+00:00 app[web.1]:     return util.import_app(self.app_uri)
2014-02-10T07:36:12.117462+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/util.py", line 354, in import_app
2014-02-10T07:36:12.116732+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2014-02-10T07:36:12.116732+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/base.py", line 114, in wsgi
2014-02-10T07:36:12.117462+00:00 app[web.1]:     __import__(module)
2014-02-10T07:36:12.117462+00:00 app[web.1]:   File "/app/rosterrun.py", line 330, in <module>
2014-02-10T07:36:12.117462+00:00 app[web.1]:     @app.route('/treasury', methods=['GET', 'POST'])
2014-02-10T07:36:12.117641+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 62, in wrapper_func
2014-02-10T07:36:12.117641+00:00 app[web.1]:     return f(self, *args, **kwargs)
2014-02-10T07:36:12.117641+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 984, in add_url_rule
2014-02-10T07:36:12.117641+00:00 app[web.1]:     'existing endpoint function: %s' % endpoint)
2014-02-10T07:36:12.117462+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 62, in load
2014-02-10T07:36:12.117462+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1013, in decorator
2014-02-10T07:36:12.117462+00:00 app[web.1]:     self.add_url_rule(rule, endpoint, f, **options)
2014-02-10T07:36:12.117641+00:00 app[web.1]: AssertionError: View function mapping is overwriting an existing endpoint function: market_results
2014-02-10T07:36:12.117641+00:00 app[web.1]: 2014-02-10 07:36:12 [7] [INFO] Worker exiting (pid: 7)
2014-02-10T07:36:12.212962+00:00 app[worker.2]: 07:36:12 *** Listening on high, default, low...
2014-02-10T07:36:12.211437+00:00 app[worker.2]: 07:36:12 RQ worker started, version 0.3.13
2014-02-10T07:36:12.212770+00:00 app[worker.2]: 07:36:12 
2014-02-10T07:36:12.272793+00:00 app[web.1]: 2014-02-10 07:36:12 [2] [INFO] Shutting down: Master
2014-02-10T07:36:12.272930+00:00 app[web.1]: 2014-02-10 07:36:12 [2] [INFO] Reason: Worker failed to boot.
2014-02-10T07:36:13.444149+00:00 heroku[web.1]: State changed from up to crashed
2014-02-10T07:36:13.445153+00:00 heroku[web.1]: State changed from crashed to starting
2014-02-10T07:36:13.431173+00:00 heroku[web.1]: Process exited with status 3
2014-02-10T07:36:23.910043+00:00 heroku[web.1]: Starting process with command `gunicorn rosterrun:app`
2014-02-10T07:36:25.710057+00:00 app[web.1]: 2014-02-10 07:36:25 [2] [INFO] Starting gunicorn 18.0
2014-02-10T07:36:25.711294+00:00 app[web.1]: 2014-02-10 07:36:25 [2] [INFO] Using worker: sync
2014-02-10T07:36:25.727328+00:00 app[web.1]: 2014-02-10 07:36:25 [7] [INFO] Booting worker with pid: 7
2014-02-10T07:36:25.711082+00:00 app[web.1]: 2014-02-10 07:36:25 [2] [INFO] Listening at: http://0.0.0.0:49343 (2)
2014-02-10T07:36:25.986087+00:00 heroku[web.1]: State changed from starting to up
2014-02-10T07:36:31.706215+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/base.py", line 114, in wsgi
2014-02-10T07:36:31.706485+00:00 app[web.1]:     __import__(module)
2014-02-10T07:36:31.706215+00:00 app[web.1]: 2014-02-10 07:36:31 [7] [ERROR] Exception in worker process:
2014-02-10T07:36:31.706215+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 495, in spawn_worker
2014-02-10T07:36:31.706215+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2014-02-10T07:36:31.706485+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 49, in load_wsgiapp
2014-02-10T07:36:31.706215+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 62, in load
2014-02-10T07:36:31.706485+00:00 app[web.1]:     return util.import_app(self.app_uri)
2014-02-10T07:36:31.706215+00:00 app[web.1]: Traceback (most recent call last):
2014-02-10T07:36:31.706215+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/workers/base.py", line 106, in init_process
2014-02-10T07:36:31.706706+00:00 app[web.1]: Traceback (most recent call last):
2014-02-10T07:36:31.706706+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/workers/base.py", line 106, in init_process
2014-02-10T07:36:31.706215+00:00 app[web.1]:     self.callable = self.load()
2014-02-10T07:36:31.706485+00:00 app[web.1]:     self.add_url_rule(rule, endpoint, f, **options)
2014-02-10T07:36:31.706485+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/util.py", line 354, in import_app
2014-02-10T07:36:31.706485+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1013, in decorator
2014-02-10T07:36:31.706706+00:00 app[web.1]:     'existing endpoint function: %s' % endpoint)
2014-02-10T07:36:31.706215+00:00 app[web.1]:     worker.init_process()
2014-02-10T07:36:31.706706+00:00 app[web.1]: AssertionError: View function mapping is overwriting an existing endpoint function: market_results
2014-02-10T07:36:31.706485+00:00 app[web.1]:     return f(self, *args, **kwargs)
2014-02-10T07:36:31.706706+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/base.py", line 114, in wsgi
2014-02-10T07:36:31.706706+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 984, in add_url_rule
2014-02-10T07:36:31.706706+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/arbiter.py", line 495, in spawn_worker
2014-02-10T07:36:31.706485+00:00 app[web.1]:     @app.route('/treasury', methods=['GET', 'POST'])
2014-02-10T07:36:31.707316+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/util.py", line 354, in import_app
2014-02-10T07:36:31.706706+00:00 app[web.1]:     self.callable = self.load()
2014-02-10T07:36:31.706706+00:00 app[web.1]:     worker.init_process()
2014-02-10T07:36:31.706215+00:00 app[web.1]:     return self.load_wsgiapp()
2014-02-10T07:36:31.707316+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 62, in load
2014-02-10T07:36:31.707316+00:00 app[web.1]:   File "/app/rosterrun.py", line 330, in <module>
2014-02-10T07:36:31.707316+00:00 app[web.1]:     __import__(module)
2014-02-10T07:36:31.706485+00:00 app[web.1]:   File "/app/rosterrun.py", line 330, in <module>
2014-02-10T07:36:31.707316+00:00 app[web.1]:     self.add_url_rule(rule, endpoint, f, **options)
2014-02-10T07:36:31.706485+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 62, in wrapper_func
2014-02-10T07:36:31.707316+00:00 app[web.1]:     return self.load_wsgiapp()
2014-02-10T07:36:31.707316+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 1013, in decorator
2014-02-10T07:36:31.706706+00:00 app[web.1]:     self.wsgi = self.app.wsgi()
2014-02-10T07:36:31.707316+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/gunicorn/app/wsgiapp.py", line 49, in load_wsgiapp
2014-02-10T07:36:31.707508+00:00 app[web.1]:     return f(self, *args, **kwargs)
2014-02-10T07:36:31.707508+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 62, in wrapper_func
2014-02-10T07:36:31.707508+00:00 app[web.1]: 2014-02-10 07:36:31 [7] [INFO] Worker exiting (pid: 7)
2014-02-10T07:36:31.707316+00:00 app[web.1]:     return util.import_app(self.app_uri)
2014-02-10T07:36:31.707508+00:00 app[web.1]:   File "/app/.heroku/python/lib/python2.7/site-packages/flask/app.py", line 984, in add_url_rule
2014-02-10T07:36:31.707508+00:00 app[web.1]:     'existing endpoint function: %s' % endpoint)
2014-02-10T07:36:31.707508+00:00 app[web.1]: AssertionError: View function mapping is overwriting an existing endpoint function: market_results
2014-02-10T07:36:31.707316+00:00 app[web.1]:     @app.route('/treasury', methods=['GET', 'POST'])
2014-02-10T07:36:31.878500+00:00 app[web.1]: 2014-02-10 07:36:31 [2] [INFO] Shutting down: Master
2014-02-10T07:36:31.878500+00:00 app[web.1]: 2014-02-10 07:36:31 [2] [INFO] Reason: Worker failed to boot.
2014-02-10T07:36:33.122181+00:00 heroku[web.1]: Process exited with status 3
2014-02-10T07:36:33.135174+00:00 heroku[web.1]: State changed from up to crashed
2014-02-10T07:36:33.609985+00:00 heroku[router]: at=error code=H10 desc="App crashed" method=GET path=/ host=rosterrun.herokuapp.com request_id=4d6a86e6-3138-477e-8090-1c29ad65f49a fwd="67.105.106.9" dyno= connect= service= status=503 bytes=
2014-02-10T07:35:59.008550+00:00 heroku[web.1]: State changed from up to starting
2014-02-10T07:35:59.060133+00:00 heroku[worker.1]: State changed from up to starting
2014-02-10T07:35:59.085305+00:00 heroku[worker.2]: State changed from up to starting
2014-02-10T07:36:09.574793+00:00 heroku[worker.1]: State changed from starting to up
